helix-term/src/commands.rs

```diff
@@ -2617,24 +2617,34 @@ fn selection_is_linewise(selection: &Selection, text: &Rope) -> bool {
     })
 }
 
-fn delete_selection_impl(cx: &mut Context, op: Operation, yank: bool) {
+enum YankMode {
```

Comment:
Small nit: let's call this `YankAction` or `YankOperation` to avoid connotation with "mode" (`Mode::{Insert, Normal, Select}`)

Reply:
Lets go with `YankAction`